### ISSUE TITLE
Dead?

### ISSUE THREAD

**Issue by discordianfish** [polvi/monokube#6]
Hi,

I currently have two uses cases where monokube would be a great fit, but it doesn't look like this is maintained anymore, correct? /cc @polvi

**Comment by philips** [comment]
Correct. This is deprecated and replaced by https://github.com/kubernetes-incubator/bootkube
